### ISSUE TITLE
Contract script is missing. Rebuild the contract

### ISSUE THREAD

**Issue by anton-trunov** [tact-lang/web-ide#49]
<img width="1002" alt="image" src="https://github.com/nujan-io/nujan-ide/assets/2316541/74098f31-7680-4861-b514-6a9af966b202">

**Comment by rahulyadav-57** [comment]
The contract name is `Foo`, but you are trying to deploy `FooMaster`. This might be due to the `FooMaster` TypeScript file being missing in the `dist` folder.

**Comment by anton-trunov** [comment]
`FooMaster` should not be in the drop-down list then, please see if you can fix this synchronization issue

**Comment by rahulyadav-57** [comment]
Yes, I think it is not getting synchronization properly. I'll look into it.
